azure/scope/cluster.go

```diff
@@ -394,6 +427,11 @@ func (s *ClusterScope) AvailabilitySetEnabled() bool {
 	return len(s.AzureCluster.Status.FailureDomains) == 0
 }
 
+// LoadBalancerNodeOutboundIPs returns the number node outbound ips that needs to be configured in the load balancer.
+func (s *ClusterScope) LoadBalancerNodeOutboundIPs() *int32 {
```

Comment:
Is it idiomatic to return the pointer type from this convenience method? Would returning an `int32` be easier for callers?

Reply:
Yeah I think it makes sense to return 1 if nil.